acceptance/tests/apply/augeas/services.rb

```diff
@@ -0,0 +1,61 @@
+test_name "Augeas services file" do
+
+  tag 'risk:medium'
+
+  confine :except, :platform => 'windows'
+  confine :except, :platform => 'osx'
+  confine :except, :platform => 'aix'
```

Comment:
We can get this to pass on AIX if we add the {{incl}} and {{lens}|} values to the {{augeas}} resources.